CorsixTH/Lua/hospital.lua

```diff
@@ -881,15 +885,15 @@ function Hospital:coldWarning()
     "sorry002.wav", "sorry004.wav",
   }
   if announcements and self:isPlayerHospital() then
-    self.world.ui:playAnnouncement(announcements[math.random(1, #announcements)])
+    self.world.ui:playAnnouncement(announcements[math.random(1, #announcements)], AnnouncementPriority.Low)
```

Comment:
I think Hot and Cold warnings should be priority Normal; save Low for the random banter?

Reply:
Yeah, I think that makes more sense. That way the heat issue can be fixed faster by the player.